### ISSUE TITLE
New release is not on bintray.

### ISSUE THREAD

**Issue by sagudev** [aranym/aranym#17]
On bintray there is no new release (except for armhf). https://dl.bintray.com/aranym/aranym-files/1.1.0/

**Comment by sagudev** [comment]
Maybe restart build. https://github.com/aranym/aranym/blob/master/.travis/build.sh#L36 probably run before you created new tag.

**Comment by th-otto** [comment]
Yes, we already noticed. I tried to restart the build, but now it fails at the very end, and i can't see any any error occurring... really strange.

**Comment by sagudev** [comment]
Did you ever test new release with Travis and git or it was now first time?

**Comment by th-otto** [comment]
It was the first time, since this is the first release since automated builds were implemented. So there might be errors in the scripts, but when i look at https://travis-ci.org/aranym/aranym/jobs/519929699 then i can't see any, still the build exits with "1" at the end.

**Comment by sagudev** [comment]
Exit code 1        General errors, Miscellaneous errors, such as "divide by zero" and other impermissible operations
maybe for command `if test -d "aranym-1.1.0"; then find "aranym-1.1.0" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "aranym-1.1.0" || { sleep 5 && rm -rf "aranym-1.1.0"; }; else :; fi`

**Comment by joysfera** [comment]
Wondering if the tag named "ARANYM_1_1_0" didn't mess with it, as the folder name is aranym-ARANYM_1_1_0. Just wild guess, didn't really inspect the scripts.

**Comment by th-otto** [comment]
But the very same build worked yesterday, when Petr created the release. Just the deploy did not work as expected. Now already the build fails when i restart it.

**Comment by sagudev** [comment]
You are right. Hm the problem is called in https://github.com/aranym/aranym/blob/master/.travis/build.sh#L172 and on first build run isrelease was not set so this code didn't get executed.

**Comment by sagudev** [comment]
It fails when running make dist. Could you try to run it locally

**Comment by th-otto** [comment]
Fixed, make dist should work again.

**Comment by sagudev** [comment]
Maybe we should bump aranym version again (1.1.0 -> 1.1.1) when we finished implementing snaps to test whole release process.
